src/store/selectors.ts

```diff
@@ -7,9 +7,13 @@ export const selectAccountAddress = (state: RootState) => state.account.address;
 export const selectUnspentAddress = (state: RootState) => state.account.unspent?.address;
 export const selectUnspentBalance = (state: RootState) => state.account.unspent?.balance;
 export const selectUnspentUtxos = (state: RootState) => state.account.unspent?.utxos || [];
-
+export const selectUnspent = (state: RootState) => state.account.unspent || [];
+export const selectChosenTransaction = (state: RootState) => state.account.chosenTx;
+export const selectTransactions = (state: RootState) =>
+  state.account.txs[state.account.address] || [];
+export const selectParsedTransactions = (state: RootState) => state.account.parsedTxs || [];
 export const selectChosenAsset = (state: RootState) => state.wallet.chosenAsset;
-export const selectAssets = (state: RootState) => state.wallet.assets;
+export const selectAssets = (state: RootState) => state.wallet.assets || [];
 export const selectCurrentTxId = (state: RootState) => state.wallet.currentTx.id;
```

Comment:
Should probably use the nullish coalescing operator (`??`) here instead of the OR operator.

Reply:
In all of them.